api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -41,6 +50,7 @@
 
     private static final boolean NOT_DUPLICATE = false;
     private static final boolean DUPLICATE = true;
+    private  static final EligibilityOutcome NO_ELIGIBILITY_OVERRIDE = null;
```

Comment:
Double space here, surprised that got through Checkstyle.

Reply:
done